Gulpfile.mjs

```diff
@@ -429,9 +429,8 @@ function buildRollup(packages, buildStandalone) {
             }),
             rollupBabel({
               envName,
-              babelrc: false,
               babelHelpers: "bundled",
-              extends: "./babel.config.js",
+              configFile: "./babel.config.js",
```

Comment:
`extends` expects a file-relative configuration, not a project-wide configuration.